### ISSUE TITLE
[BUG] Invalid chunk type

### ISSUE THREAD

**Issue by GustavsGegeris** [iBotPeaches/Apktool#2613]
### Information
1. **Apktool Version (`apktool -version`)** - 2.5.0
2. **Operating System (Mac, Linux, Windows)** - Windows 10 version 21H1
3. **APK From? (Playstore, ROM, Other)** - Play Store

### Stacktrace/Logcat

```
I: Using Apktool 2.5.0 on citymapper.apk
I: Loading resource table...
Exception in thread "main" brut.androlib.AndrolibException: Invalid chunk type: expected=0x00000200, got=0x00000203
        at brut.androlib.res.decoder.ARSCDecoder.checkChunkType(ARSCDecoder.java:544)
        at brut.androlib.res.decoder.ARSCDecoder.readTablePackage(ARSCDecoder.java:88)
        at brut.androlib.res.decoder.ARSCDecoder.readTableHeader(ARSCDecoder.java:82)
        at brut.androlib.res.decoder.ARSCDecoder.decode(ARSCDecoder.java:48)
        at brut.androlib.res.AndrolibResources.getResPackagesFromApk(AndrolibResources.java:790)
        at brut.androlib.res.AndrolibResources.loadMainPkg(AndrolibResources.java:67)
        at brut.androlib.res.AndrolibResources.getResTable(AndrolibResources.java:59)
        at brut.androlib.Androlib.getResTable(Androlib.java:66)
        at brut.androlib.ApkDecoder.setTargetSdkVersion(ApkDecoder.java:236)
        at brut.androlib.ApkDecoder.decode(ApkDecoder.java:118)
        at brut.apktool.Main.cmdDecode(Main.java:179)
        at brut.apktool.Main.main(Main.java:82)
```

### Steps to Reproduce
1. `apktool d citymapper.apk`

### Frameworks
none that i know of

### APK
https://play.google.com/store/apps/details?id=com.citymapper.app.release

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything?
2. If you are trying to install a modified apk, did you resign it?
3. Are you using the latest apktool version?

Yes.

**Comment by AndnixSH** [comment]
Duplicate https://github.com/iBotPeaches/Apktool/issues/2099

**Comment by iBotPeaches** [comment]
```
➜  2613 apktool d citymapper.apk 
I: Using Apktool 2.5.1-d1e447-SNAPSHOT on citymapper.apk
I: Loading resource table...
I: Decoding Shared Library (com.citymapper.app.release.dynamic_feature_kyc2), pkgId: 126
I: Decoding AndroidManifest.xml with resources...
I: Loading resource table from file: /home/ibotpeaches/.local/share/apktool/framework/1.apk
W: Could not decode attr value, using undecoded value instead: ns=android, name=theme, value=0x7e0d0001
W: Could not decode attr value, using undecoded value instead: ns=android, name=theme, value=0x7e0d0006
W: Could not decode attr value, using undecoded value instead: ns=android, name=resource, value=0x7e0f0001
I: Regular manifest package...
I: Decoding file-resources...
I: Decoding values */* XMLs...
I: Baksmaling classes.dex...
I: Baksmaling classes2.dex...
I: Baksmaling classes3.dex...
I: Baksmaling classes4.dex...
I: Baksmaling classes5.dex...
I: Copying assets and libs...
I: Copying unknown files...
I: Copying original files...
I: Copying META-INF/services directory
➜  2613 
```

Should disassemble fine when 2.5.1 is released. Closing as complete and the duplicate of the closed #2099 ticket above. If you try and rebuild it - you'll run into #2514 .
